pkg/util/machineset/util.go

```diff
@@ -0,0 +1,92 @@
+package util
+
+import (
+	"errors"
+)
+
+const (
+	// OpenShift introduced annotations for scaling from zero.
+	CpuKeyOriginal      = "machine.openshift.io/vCPU"
```

Comment:
could we name these like `SomethingDeprecated` instead of `SomethingOriginal` ?

Reply:
+1

Reply:
Resolved.